packages/client/tests/functional/runtime-import/tests.ts

```diff
@@ -5,7 +5,6 @@ import testMatrix from './_matrix'
 
 const libraryRuntime = 'runtime/library'
 const binaryRuntime = 'runtime/binary'
-const dataProxyRuntime = 'runtime/data-proxy'
```

Comment:
This string cannot be found in the generated client anymore, so we don't search for it anymore.